verifier/Cargo.toml

```diff
@@ -14,9 +14,8 @@ repository = { workspace = true }
 rust-version = { workspace = true }
 
 [features]
-tcb = ["dep:serde_json", "dep:serde", "dep:der", "dep:hex", "advisories", "mc-sgx-dcap-types/tcb"]
-advisories = ["dep:serde_json"]
-x509 = ["dep:mbedtls", "dep:x509-cert"]
+tcb = ["dep:serde_json", "dep:serde", "dep:der", "dep:hex", "mc-sgx-dcap-types/tcb", "x509-cert/pem"]
```

Comment:
I dropped `advisories` feature, I think it's really a part of the tcb feature, i.e. it's highly unlikely someone will want only advisories, especially since the advisories are contained in the tcb json.

Reply:
:+1:

Generally I'm against proliferation of crate features -- it frequently leads to broken builds when features are turned on in a way that isn't tested in CI, and as features proliferate you get a combinatorial explosion and you can't test them all. I usually prefer to just split things up into crates rather than have optional modules within crates, controlled by features.

In this specific case, it might be fine for the tcb feature as well to just be mandatory IMO. But that's not an issue for now